BusyList/Handlers/ReadHandler.cs

```diff
@@ -1,13 +1,25 @@
 ﻿using BusyList.Commands;
 using System;
+using System.Threading.Tasks;
+using BusyList.Parsing;
+using Sprache;
 
 namespace BusyList.Handlers
 {
     internal class ReadHandler : IHandler<ReadCommand>
     {
+        private readonly ITaskRepository _taskRepository;
+
+        public ReadHandler(ITaskRepository taskRepository)
+        {
+            _taskRepository = taskRepository;
+        }
+
         public void Run(ReadCommand command)
         {
-            Console.WriteLine($"If the ReadHandler was implemented, I would now list details for Task {command.Id}.");
+            //TODO: Add logic to make this function run correctly
+            var task = _taskRepository.GetTaskById(command.Id);
+            Console.WriteLine($"If the ReadHandler was implemented, I would now list details for Task {task}.");
```

Comment:
You should be able to implement this with whats currently in the codebase. And if not, this should be a draft PR.